package.json

```diff
@@ -193,20 +193,23 @@
         "mindaro-dev.file-downloader"
     ],
     "devDependencies": {
+        "@types/glob": "^7.1.1",
+        "@types/got": "^9.6.12",
```

Comment:
Removed deprecated dependency ```request``` which is used to make ```http``` calls and replaced with ```got```. More info [here](https://github.com/request/request/issues/3142)